client/src/components/R&R/ReviewsBreakdown.jsx

```diff
@@ -5,11 +5,84 @@ import { ProductContext } from '../../context';
 
 const ReviewsBreakdown = () => {
   const { reviews, reviewsMeta, reviewSort, setReviewSort } = useContext(ProductContext);
+  const options = {
+    Size: ['A size too small', 'Perfect', 'A size too big'],
+    Width: ['Too narrow', 'Perfect', 'Too wide'],
+    Comfort: ['Uncomfortable', 'Ok', 'Perfect'],
+    Quality: ['Poor', 'What I expected', 'Perfect'],
+    Length: ['Runs short', 'Perfect', 'Runs long'],
+    Fit: ['Runs tight', 'Perfect', 'Runs long'],
+  };
+
+  const recommendPercentage = () => {
+    const total = Number(reviewsMeta.recommended.true) + Number(reviewsMeta.recommended.false);
+    return Math.round((reviewsMeta.recommended.true / total) * 100);
```

Comment:
Since you're doing this in a few places, it might be useful to have a `getPercent` function.